providers/google/src/airflow/providers/google/cloud/hooks/cloud_storage_transfer_service.py

```diff
@@ -712,3 +714,18 @@ async def operations_contain_expected_statuses(
                 f"Expected: {', '.join(expected_statuses_set)}"
             )
         return False
+
+    async def run_transfer_job(self, job_name: str) -> operation_async.AsyncOperation:
```

Comment:
@Crowiant I think the return type is incorrect here, I see that the code returns `response` which should have the result of `operation` not the operation itself, correct me if I am wrong?